src/main/java/redis/clients/jedis/JedisSentinelPool.java

```diff
@@ -129,14 +139,17 @@ public JedisSentinelPool(String masterName, Set<String> sentinels,
       final int sentinelConnectionTimeout, final int sentinelSoTimeout, final String sentinelPassword,
       final String sentinelClientName) {
     this(masterName, sentinels, poolConfig, connectionTimeout, soTimeout, null, password, database, clientName,
-        sentinelConnectionTimeout, sentinelSoTimeout, null, sentinelPassword, sentinelClientName);
+        sentinelConnectionTimeout, sentinelSoTimeout, null, sentinelPassword, sentinelClientName, false, false,
+        null, null, null);
   }
 
   public JedisSentinelPool(String masterName, Set<String> sentinels,
       final GenericObjectPoolConfig poolConfig, final int connectionTimeout, final int soTimeout,
       final String user, final String password, final int database, final String clientName,
       final int sentinelConnectionTimeout, final int sentinelSoTimeout, final String sentinelUser,
-      final String sentinelPassword, final String sentinelClientName) {
```

Comment:
This removes an existing constructor. We can't do that.

Hint: Keep this constructor, copy previous constructor from earlier code and paste it just before this constructor.

Reply:
restored the missing constructor